contents/data-display/cards/payment-method/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { CreditCardIcon } from "@yamada-ui/lucide"
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  FormControl,
+  HStack,
+  Heading,
+  Input,
+  Text,
+  useRadioGroup,
+  Autocomplete,
+  AutocompleteOption,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+import { AppleIcon } from "./apple"
+import { CustomRadio } from "./custom-radio"
+import { PaypalIcon } from "./paypal"
+
+const PaymentMethod: FC = () => {
+  const { getContainerProps, getRadioProps } = useRadioGroup<string>({
+    defaultValue: "Card",
+  })
+
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ]
+
+  const currentYear = new Date().getFullYear()
+  const years = Array.from({ length: 10 }, (_, i) => currentYear + i)
+
+  return (
+    <Card maxW="md" rounded="xl" variant="outline" as="form">
```

Comment:
Please add `preventDefault()`

```suggestion
    <Card maxW="md" rounded="xl" variant="outline" as="form" onSubmit={e => e.preventDefault()} >
```

Reply:
Fixed.